data/fields/barrier.json

```
@@ -1,5 +1,32 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
+             "bollard": "Bollard",
+             "lift_gate": "Boom Barrier",
```

```suggestion
             "lift_gate": "Boom Gate",
```

to keep it similar to other types of gates.